### ISSUE TITLE
No label if name is prohibited?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/accname#99]
There are [several roles](https://www.w3.org/TR/wai-aria-1.2/#namefromprohibited) where explicit names are prohibited. If they do have one (aria-label, aria-labelledby): should it be ignored? If so, this should be mentioned in 2B and 2C

**Comment by jnurthen** [comment]
This is covered by https://github.com/w3c/accname/pull/53
